src/components/molecules/Dashboard/Workspace.tsx

```diff
@@ -30,11 +33,20 @@ const Workspace: React.FC<Props> = ({ className, team }) => {
         </Text>
         <Flex>
           <TeamWrapper flex={4}>
-            {team?.members?.map((member, i) => (
-              <Text key={i} size="m" color={theme.main.text}>
-                {member?.user.name}
-              </Text>
+            {team?.members?.slice(0, 5).map((member, i) => (
+              <StyledAvatar key={i} size={32} color={theme.main.avatarbg} radius={50}>
+                <Text size={isSmallWindow ? "m" : "l"} color={theme.text.pale}>
+                  {member?.user.name?.charAt(0).toUpperCase()}
+                </Text>
+              </StyledAvatar>
             ))}
+            {teamLength > 5 && (
+              <StyledAvatar size={32} color={theme.main.avatarbg} radius={50}>
+                <Text size={isSmallWindow ? "m" : "l"} color={theme.text.pale}>
+                  +{teamLength - 5}
+                </Text>
+              </StyledAvatar>
+            )}
```

Comment:
Please see the comment on the `Avatar` component about changes to it.
For this code in particular:

This is fine, but I feel like if you use slice like this it isn't very clear why, or for what reason. 
If you can, one option would be instead of the slice here and declaring `teamLength` above, perhaps calculate `excessMembers`, and use that for filtering and conditionally rendering the `+` avatar.
Example:
```
const excessMembers = team?.members?.length - 5 ?? 0;

const shownMembers = useCallback(()=> ....return only first 5 members... );

return (
...
{shownMembers?.map((member, i) => (
   <Avatar key={i} value={member) />
)}
{excessMembers && (
 <Avatar value={excessMembers} /> 
)}
```